File: textgen.js

var tech = require('./tech_vars.js');
var verbs = require('./verbs_vars.js');
var issues = require('./issues_vars.js');

function getRandomInt(min, max) {
  min = Math.ceil(min);
  max = Math.floor(max);
  return Math.floor(Math.random() * (max - min)) + min;
}

function choise(arr){
  var rndd = Math.floor(Math.random()*arr.length);
  return arr[rndd];
}

var i_tech = getRandomInt(0, tech.length)
var i_verbs = getRandomInt(0, verbs.length)
var i_issues = getRandomInt(0, issues.length)
console.log(i_tech,i_verbs,i_issues );
console.log(choise(tech[i_tech]),choise(verbs[i_verbs]),choise(issues[i_issues]) );

File: textgen.py
'''https://docs.google.com/spreadsheets/d/11Ter3a38OXjwzwkWccF96ng3CFvFEx-bPBR__1IPh1Q/edit?usp=sharing'''

import sys
import random
def loadFromFile(fn):
    with open(fn) as textFile:
        list = [line.strip(' \t\n\r').split(",") for line in textFile]
    return list

tech=loadFromFile("tech.txt")
verb=loadFromFile("verbs.txt")
issues=loadFromFile("issues.txt")

i_tech=random.randint(0, len(tech))
i_verb=random.randint(0, len(verb))
i_issues=random.randint(0, len(issues))

print
print
print random.choice(tech[i_tech]),random.choice(verb[i_verb]),random.choice(issues[i_issues])
print
print
